### DISCUSSION TITLE
/mnt/wslg directory is not appearing

### DISCUSSION THREAD

**Discussion by kasvith** [microsoft/wslg#921] [Q&A] [answered]
Hi, I have reinstalled WSL2 from the Store with the Ubuntu distro and followed the guide to get the update, enabling GUI apps with the relevant latest NVIDIA Drivers.

But currently, I don't see a `/mnt/wslg` directory, hence i don't see `/tmp/.X11` directory too.

How can i get this work?

**System Info**:
```
OS Name             : Microsoft Windows 10 Education
OS Version          : 10.0.19044 N/A Build 19044
OS Manufacturer     : Microsoft Corporation
OS Configuration    : Standalone Workstation
OS Build Type       : Multiprocessor Free
System Boot Time    : 12/3/2022, 9:52:14 AM
System Manufacturer : ASUSTeK COMPUTER INC.
System Model        : GL502VSK
System Type         : x64-based PC
System Directory    : C:\WINDOWS\system32
System Locale       : en-gb;English (United Kingdom)
Hotfix(s)           : 26 Hotfix(s) Installed.,[01]: KB5020613,[02]: KB4537759,[03]: KB4557968,[04]: KB4561600,[05]:
                      KB4562830,[06]: KB4566785,[07]: KB4570334,[08]: KB4577266,[09]: KB4577586,[10]: KB4580325,[11]:
                      KB4586864,[12]: KB4589212,[13]: KB4593175,[14]: KB4598481,[15]: KB5000736,[16]: KB5003791,[17]:
                      KB5012170,[18]: KB5019959,[19]: KB5006753,[20]: KB5007273,[21]: KB5014035,[22]: KB5014671,[23]:
                      KB5015895,[24]: KB5016705,[25]: KB5018506,[26]: KB5005699
```

**Comment by erdometo** [comment]
I have the same problem. Tried disabling WSL and reinstalling. I also tried installing wsl from mstore.
 /mnt/wslg is not showing up, but somehow i am able to run Nvidia Cuda demos.

Could it be related with secureboot and memory encryption?

**Reply by Biswa96** [reply]
Is there any option in .wslconfig or wsl.conf which can disable the gui feature?

**Reply by erdometo** [reply]
I set gui enable option to true.

**Comment by kasvith** [comment, marked as answer]
I managed to solve this by updating windows, there were few feature updates
